#! /bin/sh /usr/share/dpatch/dpatch-run
## 70-fix_readme_link.dpatch by Benjamin Seidenberg <astronut@dlgeek.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix a bad link in readme.html

@DPATCH@
--- a/doc/readme.html
+++ b/doc/readme.html
@@ -56,7 +56,7 @@ This section reserved for WARNING WARNIN
 <P> This system should be expected to have the same order-of-magnitude
 installation complexity as a netnews system.  Maintenance should have
 similar complexity, except administrators will have to deal with <A
-HREF="cyradm.1.html">creation and deletion of users</A> and will have
+HREF="man/cyradm.1.html">creation and deletion of users</A> and will have
 the option of managing quotas and access control lists.
 
 <p>This package is distributed as a source only release. It is
